### PR TITLE
fix checking of uploaded samples

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

N.A.

#### Link to staging deployment URL 

https://ui-agi-test-pipeline.scp-staging.biomage.net/

#### Anything else the reviewers should know about the changes here

- The previous function checks for the order of the file uploads, the new function just checks if the required files exists

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [X] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/components/data-management/ProjectDetails.jsx [hunk 1]
```diff
@@ -481,10 +481,11 @@ const ProjectDetails = ({ width, height }) => {
       // Check if all files for a given tech has been uploaded
       const fileNamesArray = Array.from(sample.fileNames);
 
-      if (!_.isEqual(
```
Comment (aerlaut) [inline src/components/data-management/ProjectDetails.jsx:484]: `_.isEqual` checks for order. `fileNamesArray` may not be uploaded in the same order as `fileUploadSpecifications[sample.type].requiredFiles` hence it fails.

File: src/pages/data-management/index.jsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ const DataManagementPage = ({ route }) => {
   const unnamedExperimentName = 'Unnamed Analysis';
 
   const createNewProject = (newProjectName, newProjectDescription) => {
-    const numUnnamedExperiments = !existingExperiments[0] ? 0
+    const numUnnamedExperiments = !existingExperiments ? 0
```
Comment (aerlaut) [inline src/pages/data-management/index.jsx]: `null` value doesn't have `[0]`

Comment (cosa65) [inline src/pages/data-management/index.jsx]: Change to ?.[0] to keep it closer to original

Comment (aerlaut) [reply src/pages/data-management/index.jsx]: Changed

**Review by cosa65 — APPROVED** [review]

**Pull request merged**